### PR TITLE
lightning: init client-go global cfg

### LINKED ISSUE #45462: lightning checksum report "Region is unavailable" and  tikv report "Handshake failed with fatal error SSL_ERROR_SSL: error:1408F10B:SSL routines:ssl3_get_record:wrong version number"
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)
- use below config to make it easier to reproduce
```toml
checksum-table-concurrency = 1
distsql-scan-concurrency = 5 # or even smaller
```
-  import 2 tables. checksum each table takes about 10m
- during checksum of second table will report error

related issue https://github.com/tikv/grpc-rs/issues/619

### 2. What did you expect to see? (Required)
success
### 3. What did you see instead (Required)
checksum failed
### 4. What is your TiDB version? (Required)
7.1.0/6.5.2

### PR DESCRIPTION
**Pull request by D3Hunter**
### What problem does this PR solve?

Issue Number: close #45462 

Problem Summary:

### What is changed and how it works?
- init client-go global cfg
### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
  - tested twice on a data-set which can reproduce after init client-go global cfg, import success
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
fix lightning checksum report "Region is unavailable" when target cluster enabled SSL
```

### CODE REVIEW

File: br/pkg/lightning/importer/import.go [hunk 1]
```diff
@@ -2227,3 +2230,18 @@ func filterColumns(columnNames []string, extendData mydump.ExtendColumnData, ign
 	}
 	return filteredColumns, extendValueDatums
 }
+
+// check store liveness of tikv client-go requires GlobalConfig to work correctly, so we need to init it,
+// else tikv will report SSL error when tls is enabled.
+// and the SSL error seems affects normal logic of newer TiKV version, and cause the error "tikv: region is unavailable"
+// during checksum.
+// todo: DM relay on lightning physical mode too, but client-go doesn't support passing TLS data as bytes,
+func initGlobalConfig(secCfg tikvconfig.Security) {
```
Comment (lance6716) [inline br/pkg/lightning/importer/import.go:2239]: Can we move it to an earlier place? I'm not sure where's the first usage of tikv client.

**Review by lance6716 — COMMENTED** [review]
rest lgtm

File: br/pkg/lightning/importer/import.go (hunk 1 above)
Comment (D3Hunter) [reply br/pkg/lightning/importer/import.go:2240]: https://github.com/pingcap/tidb/pull/45464/commits/0581febebfe855a50f7e6d90d3c56645dadccac9

**Pull request merged**